app/src/main/kotlin/co/netguru/baby/monitor/client/feature/analytics/UserProperty.kt

```diff
@@ -0,0 +1,17 @@
+package co.netguru.baby.monitor.client.feature.analytics
+
+import co.netguru.baby.monitor.client.data.splash.AppState
+import co.netguru.baby.monitor.client.feature.voiceAnalysis.VoiceAnalysisOption
+
+sealed class UserProperty(val value: String, val key: String) {
+    class VoiceAnalysis(voiceAnalysisOption: VoiceAnalysisOption) :
+        UserProperty(voiceAnalysisOption.name.toLowerCase(), VOICE_ANALYSIS_KEY)
+
+    class AppStateProperty(appState: AppState) :
+        UserProperty(appState.name.toLowerCase(), APP_STATE_KEY)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/dd68c2ed2c957a6e8bc85bec93f0f874e01caa7c/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/analytics/UserProperty.kt#L11"></span>Implicitly using the default locale is a common source of bugs: Use <code>toLowerCase(Locale)</code> instead. For strings meant to be internal use <code>Locale.ROOT</code>, otherwise <code>Locale.getDefault()</code>.</td>
    </tr>
  </tbody>
</table>